cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -95,6 +98,14 @@ func (command *Evaluate) SetLogger(logger *log.Logger) {
 	command.logger = logger
 }
 
+var _ SetArguments = (*Evaluate)(nil)
+
+// SetArguments sets the commands arguments.
+func (command *Evaluate) SetArguments(args []string) {
+	availableFlags := util.Flags(command)
+	command.args = util.FilterArgsKeep(args, availableFlags)
+}
```

Comment:
cmd/eval-dev-quality/cmd/command.go

Reply:
So all of this is basically to avoid `os.Args[2:]`?

Reply:
Yes because `os.Args` contains all the flags used for testing too etc. and because of that it was not possible to get a clear slice of the parameters which should be passed down to the container image.